modules/sharedaddy/admin-sharing.js

```diff
@@ -123,7 +123,12 @@
 				$( '#live-preview div.sharedaddy' ).addClass( 'sd-social-icon' );
 			} else if ( 'official' === button_style ) {
 				$( '#live-preview ul.preview .advanced, .sharing-hidden .inner ul .advanced' ).each( function( /*i*/ ) {
-					if ( !$( this ).hasClass( 'preview-press-this' ) && !$( this ).hasClass( 'preview-email' ) && !$( this ).hasClass( 'preview-print' ) && !$( this ).hasClass( 'share-custom' ) ) {
+					if ( !$( this ).hasClass( 'preview-press-this' )
+						&& !$( this ).hasClass( 'preview-email' )
```

Comment:
`&&` needs to go at the end of the previous line so jshint doesn't complain, I think the idea of this rule is to make it clear by the end of the line whether the expression ends there or could continue on the next line.

I don't really like this enforcement but it is why the tests are failing.